api/src/main/java/org/motechproject/nms/api/web/contract/mobileAcademy/SaveBookmarkRequest.java

```diff
@@ -0,0 +1,49 @@
+package org.motechproject.nms.api.web.contract.mobileAcademy;
+
+import java.util.Map;
+
+/**
+ * Bookmark request object to save a bookmark 2.2.5
+ */
+public class SaveBookmarkRequest {
+
+    private Long callingNumber;
+
+    private Long callId;
+
+    private String bookmark;
+
+    private Map<String, Integer> scoresByChapter;
+
```

Comment:
I think it's nice to explicitly define the nullary (parameter-less) constructor, as you've done for most of the contract classes, even if the java compiler will generate one for you. Also a reminder that Jackson uses this constructor to convert the request JSON to your request object.